inference-engine/tests/functional/plugin/shared/include/single_layer_tests/one_hot.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+
+using oneHotParams = typename std::tuple<
+    int64_t,                    // axis
+    size_t,                     // depth
+    float,                      // on_value
+    float,                      // off_value
+    std::vector<size_t>,        // Input shapes
+    InferenceEngine::Precision, // Net precision
+    LayerTestsUtils::TargetDevice // Target device name                   >
```

Comment:
Stray whitespace and `>` at the end of this line

Reply:
Will remove them in futher PR.